src/test/java/redis/clients/jedis/mocked/unified/UnifiedJedisBitmapCommandsTest.java

```diff
@@ -0,0 +1,368 @@
+package redis.clients.jedis.mocked.unified;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.sameInstance;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.Test;
+import redis.clients.jedis.args.BitCountOption;
+import redis.clients.jedis.args.BitOP;
+import redis.clients.jedis.params.BitPosParams;
+
+public class UnifiedJedisBitmapCommandsTest extends UnifiedJedisTestBase {
```

Comment:
You are putting these in unified package. So something other than unified should start these classes names. For example: `<DataStructure>MockedTest`.